### PR TITLE
bce-iot-9474: add node85 doc of setup

### PR DESCRIPTION
**Pull request by chensheng0**

### CODE REVIEW

File: doc/us-en/setup/Install-OpenEdge-on-CentOS.md [hunk 1]
```diff
@@ -46,31 +46,50 @@ docker version
 
 **For more details, please see the [official documentation](https://docs.docker.com/install/).**
 
-### Install Python and runtime dependency package in **native** process mode
+### Install related runtime environment and its dependencies in **native** process mode
```
Comment (kuailehaibin) [inline doc/us-en/setup/Install-OpenEdge-on-CentOS.md]: 太长了标题，缩短一下，Install prerequisites

File: doc/us-en/setup/Install-OpenEdge-on-CentOS.md [hunk 2]
```diff
@@ -46,31 +46,50 @@ docker version
...
 
-OpenEdge provides Python Runtime, which supports running code written in Python2.7 and Python3.6. If you plan to use the **native** process mode to start, it is recommended to install **Python3.6** or higher version locally and run the package it depends on. If you already have other version of Python3 lower than 3.6, it is recommended that you uninstall it first and install Python3.6. Or you can keep the inconsistent version but need to ensure compatibility.
+OpenEdge provides the Python runtime and the Node runtime. If you plan to use the **native** process mode, you need to install these runtime environments and dependencies locally. The corresponding versions are Python2.7, Python3.6, and Node8.5. Users can also choose other versions, but they need to ensure compatibility.
```
Comment (kuailehaibin) [inline doc/us-en/setup/Install-OpenEdge-on-CentOS.md]: 不用特指了吧，去掉冠词 the。OpenEdge provides Python and Node runtime. If you plan to use the **native** process mode, you need to install prerequisites first. The corresponding versions for Python runtime are Python2.7 and Python3.6, and Node8.5 for Node runtime. Besides, users can also choose other versions, but they must ensure compatibility.

File: doc/us-en/setup/Install-OpenEdge-on-CentOS.md [hunk 3]
```diff
@@ -46,31 +46,50 @@ docker version
...
-- Step 3：Install the packages need by OpenEdge based on Python3.6.
+- Step 3：Install dependencies required by OpenEdge:
 
 ```shell
 sudo pip3 install grpcio protobuf pyyaml
 ```
 
+#### Install Node runtime
+
+- Step 1：Check to see if Node8.5 or above is already installed. If not go directly executing Step 2.
```
Comment (kuailehaibin) [inline doc/us-en/setup/Install-OpenEdge-on-CentOS.md]: Step 1：Check Node8.5 or above is already installed or not. If not, goto Step 2.

File: doc/zh-cn/setup/Install-OpenEdge-on-CentOS.md [hunk 4]
```diff
@@ -47,17 +47,21 @@ docker version
 
 **更多内容请参考 [官方文档](https://docs.docker.com/install/)。**
 
-### 进程模式下安装 Python 及其依赖包
+### 进程模式下安装相关运行时环境及其依赖
 
-OpenEdge 提供了 Python 运行时，支持 Python2.7 版本、Python3.6 版本的运行。如计划使用 **native** 进程模式启动，这里推荐本地安装 Python3.6 或以上版本及运行所依赖的包。推荐安装 Python3.6 或以上版本，如果已经存在 Python3.6 以下版本，可以选择卸载后重装 Python3.6，也可以选择保留用户需要自行保证代码兼容 Python3.6 以保证可以正常使用函数计算服务。
+OpenEdge 提供了 Python 运行时、Node运行时。如计划使用 **native** 进程模式启动，需要本地安装这些运行时环境及其相关依赖。对应版本分别为 Python2.7、Python3.6、Node8.5。用户也可以选择其他版本，但需要自行保证兼容性。
```
Comment (kuailehaibin) [inline doc/zh-cn/setup/Install-OpenEdge-on-CentOS.md]: 英文与中文之间加空格，Node 运行时

File: doc/us-en/setup/Install-OpenEdge-on-CentOS.md (hunk 1 above)
Comment (chensheng0) [reply doc/us-en/setup/Install-OpenEdge-on-CentOS.md]: done, us and ch

File: doc/us-en/setup/Install-OpenEdge-on-CentOS.md (hunk 2 above)
Comment (chensheng0) [reply doc/us-en/setup/Install-OpenEdge-on-CentOS.md]: done

File: doc/us-en/setup/Install-OpenEdge-on-CentOS.md (hunk 3 above)
Comment (chensheng0) [reply doc/us-en/setup/Install-OpenEdge-on-CentOS.md]: done, same as Python3.6

File: doc/zh-cn/setup/Install-OpenEdge-on-CentOS.md (hunk 4 above)
Comment (chensheng0) [reply doc/zh-cn/setup/Install-OpenEdge-on-CentOS.md]: done

**Pull request merged**